GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/HealthTests.cs

```diff
@@ -240,10 +240,10 @@ private void ValidateSubDirectoryHealth(List<string> outputLines, List<string> s
             {
                 // Regex to extract the most hydrated subdirectory names and their hydration percentage
                 // "  <hydrated-file-count> | <directory-name>" listed several times for different directories

```

Comment:
I think this example text needs to be updated?

Reply:
Yes - good catch